lib/pascal.rb

```diff
@@ -1,36 +1,33 @@
+class Pascal_triangle
+  def draw(height=1)
+    spaces = " "
```

Comment:
When you add an `initialize` method `space` can be defined there also, with proper getter method of course.